rfcs/2021-09-08-motion-design-easing.md

```diff
@@ -0,0 +1,151 @@
+| Flutter| standardEasing| 
+| iOS| [[CAMediaTimingFunction alloc] <br> initWithControlPoints:0.4f:0.0f:0.2f:1.0f] | 
+| After effects | outgoing 20%, incoming 62% | 
+
+
+# Motivation
+### Why are we doing this?
+Easing is one of the elements of motion (out of 3 core actions, duration).
+ 
+* Strictly linear movement appears unnatural to the human eye and we are trying to make it pleasing to users eyes.
+* To maintain the consistency & branding throughout our motion styles so that it looks familiar to users and doesn't look out of the world. Defining these properties will help us in guiding the designers and communicating with the developers. 
+* Motion helps users to reach their goals and makes it self-explanatory, and these easing curves will help us in achieving the motion stylesand self-explanatory motion hints.
+* Defining the easing styles will help in guiding designers and communicating with developers.
+
+
+### What is the expected outcome?
+We shall be able to answer following questions as a conclusion to this RFC
+1. What easings we'll be using for any motion?
+2. How will we ensure accessibility in terms of motion?
+3. How all easings will be used to elements or components.
+4. How/where we will store the different easings as design tokens?
+
+# Detailed Design
+## Easing Types
+Based on motion purposes we need 4 types of easings-
+
+1. **Effective:** This Easing is appropriate for moments when the user needs to focus on completing tasks. Microinteractions in Carbon such as button states, dropdowns, revealing additional information, or rendering data tables and visualizations were all designed with Effective easing. It should be quicker.
+
+2. **Reveling:** for significant moments such as opening a new page, clicking the primary action button, or when the movement itself conveys a meaning which builds self-explanatory motion hints. It can be more expressive
+
+3. **Attentive:** This Easing helps in getting user attention when the user is confused or needs guidance or information. It should get user attention but don’t distract.
+
+4. **Wary:** Easing which guide user about user or system errors, it should get instant user attention. It can be shaky.
+
+## Design Approach:
+1. Setting up Brand and word list.   ->Checkout [Razorpay Brand Document](https://razorpay.slack.com/files/UETR880K1/F01AMED4J6P/razorpay___branding_document_10.10.2020.pdf). 
+2. With the help of the few brand defining words, we designed Few effective easings. 
+3. Experimented with effective easing style concepts. 
+4. Designed Other 3 easings based on the decided concept. 
+- Final concept: Pick up Quick in starting like Racing car starts and the smooth ending which build trust. <img src="images/motion/easing/image9.png" alt="effectice-entrance" width="200" >
+* Check out the [Process](https://www.figma.com/proto/iIbMKg31KfQTlpkxwlaBXG/Motion-design-system?page-id=1270%3A0&node-id=5937%3A3235&viewport=-760%2C188%2C0.04553482308983803&scaling=scale-down) here.
```

Comment:
the link is says prototype not found

Reply:
Will update, here is the link for now. [Motion Branding Process](https://www.figma.com/proto/i8SARuAQ4EHbA4CD1PuCIj/Motion-Research-and-Ideation?page-id=2%3A1260&node-id=2%3A2290&viewport=290%2C48%2C0.07&scaling=scale-down&starting-point-node-id=2%3A2290&show-proto-sidebar=1)